e2e/cypress/support/commands.js

```diff
@@ -75,3 +75,23 @@ Cypress.Commands.add("cleanupUsers", (id_token) => {
     }
   })
 })
+
+Cypress.Commands.add("cleanupProjects", (id_token) => {
+  cy.request({
+    auth: { bearer: id_token },
+    method: 'GET',
+    url: `/apis/iam/v2beta/projects`,
+    failOnStatusCode: false
```

Comment:
[nit] Do we need this? Which non-success status codes are the ones we expect and want to ignore?

Reply:
I think we're doing this bc we don't want to fail on conflict (for create) or 404 (for delete). I assume Brenna was encountering conflicts on reruns of the tests? We could maybe delete them . . .

Reply:
I'll ask when she gets back since I'm maybe missing context . . .that sound ok?